resources/cli.conf

```diff
@@ -10,7 +10,10 @@
 Connection options|p|port|PORT|number|optional|PostgreSQL database server port (default: "5432")
 Connection options|d|dbname|DBNAME|word|optional|database name to connect to (default: "postgres")
 Connection options|U|username|USERNAME|word|optional|database user name (default: current)
-Connection options|h|hostname|HOST|alnum|mandatory|database and ssh server host or socket directory (default: "local socket")
+Connection options|h|hostname|HOST|text|mandatory|database and ssh server host
+Connection options|s|pg-socket-dir|PGSOCKET|text|optional|PostgreSQL domain socket dir (default: psql's default)
```

Comment:
there is light inconsistency in naming here -- various prefixes (`DB...`, `PG...`, and no prefix as in `HOST`)